### ISSUE TITLE
App crashes when trying to load pictures 

### ISSUE THREAD

**Issue by zaziemo** [rubymonsters/speakerinnen_liste#1064]
check what is wrong:
There are different kinds of errors we got into our devops account

`An ActionView::Template::Error occurred in profiles#show`
`MiniMagick::Invalid occurred in representations#show`

**Comment by zaziemo** [comment]
@sabrina-ulbrich do you have an idea what happens?

**Comment by sabrina-ulbrich** [comment]
@zaziemo I don't know, something is wrong with the pictures, but I have no idea how to find out what's wrong with it.

**Comment by sabrina-ulbrich** [comment]
@zaziemo maybe look in the database for `image.blob.content_type` for the concerned profiles. We have to check that no webp images are uploaded, that can cause errors.

**Comment by sabrina-ulbrich** [comment]
@zaziemo I checked all mails with this error message and all these errors were caused by one and the same profile picture. This picture was empty, 0 kb. I removed it, because every time a page loaded with this empty image the error occured with a bad looking placeholder. I'll open a PR to exclude picture formats other than jpg, gif or png, because if someone tries to upload a webp image it is in the database but can not shown and this leads to another error.
